package.json

```diff
@@ -19,7 +19,7 @@
     "deduplicate": "node scripts/deduplicate.js",
     "l10n": "babel-node -x .ts ./scripts/l10n.ts",
     "stylelint": "stylelint '**/*.js' '**/*.ts' '**/*.tsx'",
-    "prettier": "node ./scripts/prettier.js --branch next",
+    "prettier": "node ./scripts/prettier.js --branch master",
```

Comment:
master is the default but I'm forcing it in case the main mono-repo changes its base branch.

Reply:
We should add a `Switch main branch` paragraph in the `scripts` README.
We already forgot to switch that one when doing `master`  =>  `next` a few months ago.

Reply:
Yeah, OK, doing a new PR for this.